pa11ycrawler/pipelines.py

```diff
@@ -109,20 +109,17 @@ def write_pa11y_config(self, item):
         config_file.close()
         return config_file
 
-    def check_title_match(self, expected_title, pa11y_output, logger):
+    def check_title_match(self, expected_title, pa11y_results, logger):
         """
         Check if Scrapy reports any issue with the HTML <title> element.
         If so, compare that <title> element to the title that we got in the
         A11yItem. If they don't match, something is screwy, and pa11y isn't
         parsing the page that we expect.
         """
-        if not pa11y_output:
-            # no output from pa11y, nothing to check.
-            return
-        pa11y_results = json.loads(pa11y_output.decode('utf8')).get("results")
         if not pa11y_results:
+            # no output from pa11y, nothing to check.
             return
-        title_errs = [err for err in pa11y_results
+        title_errs = [err for err in pa11y_results.get("results", [])
```

Comment:
do we need to decode? I do recall there being ascii errors with page titles in a previous version of this tool...